libs/design-system/social-media-button/src/lib/social-media-button.component.html

```diff
@@ -0,0 +1,3 @@
+<a [href]="links[name()]" target="_blank" rel="noopener noreferrer">
```

Comment:
Use computed properties instead of complicated computations in the html template